auth-api/src/auth_api/services/org.py

```diff
@@ -227,65 +228,84 @@ def _create_payment_settings(
-            pay_request["paymentInfo"]["bankInstitutionNumber"] = payment_info.get("bankInstitutionNumber", None)
-            pay_request["paymentInfo"]["bankAccountNumber"] = payment_info.get("bankAccountNumber", None)
-            pay_request["padTosAcceptedBy"] = kwargs["user_context"].user_name
-        # invoke pay-api
-        token = RestService.get_service_account_token()
-        if is_new_org:
-            response = RestService.post(
-                endpoint=f"{pay_url}/accounts", data=pay_request, token=token, raise_for_status=True
-            )
-        else:
-            response = RestService.put(
-                endpoint=f"{pay_url}/accounts/{org_model.id}", data=pay_request, token=token, raise_for_status=True
-            )
-
-        if response.status_code == HTTPStatus.OK:
-            payment_account_status = PaymentAccountStatus.CREATED
-        elif response.status_code == HTTPStatus.ACCEPTED:
-            payment_account_status = PaymentAccountStatus.PENDING
-        else:
-            payment_account_status = PaymentAccountStatus.FAILED
+            if payment_method == PaymentMethod.PAD.value:  # PAD has bank related details
+                pay_request["paymentInfo"]["bankTransitNumber"] = payment_info.get("bankTransitNumber", None)
+                pay_request["paymentInfo"]["bankInstitutionNumber"] = payment_info.get("bankInstitutionNumber", None)
+                pay_request["paymentInfo"]["bankAccountNumber"] = payment_info.get("bankAccountNumber", None)
+                pay_request["padTosAcceptedBy"] = kwargs["user_context"].user_name
+            # invoke pay-api
+            token = RestService.get_service_account_token()
+            if is_new_org:
+                response = RestService.post(
+                    endpoint=f"{pay_url}/accounts", data=pay_request, token=token, raise_for_status=True
+                )
+            else:
+                response = RestService.put(
+                    endpoint=f"{pay_url}/accounts/{org_model.id}", data=pay_request, token=token, raise_for_status=True
+                )
 
-        if payment_account_status != PaymentAccountStatus.FAILED and payment_method:
-            payment_method_description = (
-                PaymentMethod(payment_method).name if payment_method in [item.value for item in PaymentMethod] else ""
-            )
-            ActivityLogPublisher.publish_activity(
-                Activity(
-                    org_model.id,
-                    ActivityAction.PAYMENT_INFO_CHANGE.value,
-                    name=org_model.name,
-                    value=payment_method_description,
+            if response.status_code == HTTPStatus.OK:
+                payment_account_status = PaymentAccountStatus.CREATED
+            elif response.status_code == HTTPStatus.ACCEPTED:
+                payment_account_status = PaymentAccountStatus.PENDING
+            elif response.status_code == HTTPStatus.CREATED:
```

Comment:
Maybe a comment to explain this?

![image](https://github.com/user-attachments/assets/08e06f0d-a625-4070-a980-13180fd36072)

Reply:
I think in TEST, it's possible CFS account status isn't PENDING and it's ACTIVE right away if the holding date for pad is set to -1 on DEV/TEST but 3 on PROD

Reply:
201 is from unit test, it has around 290 unit tests return 201 here.

Reply:
I removed it, I throw business exception here before so lots of unit tests failed, I think it can be removed now cause I changed it to logger.warning